src/cdk/bootstrap.ts

```diff
@@ -364,4 +371,75 @@ export class MiraBootstrap {
   async undeploy (): Promise<void> {
     return await this.deploy(true)
   }
+
+  /**
+   * Changes context to use dev config if available, and runs passed function.
+   * Typical usecase for this function is to set Dev environment in the context of Mira executable.
+   * In case of CDK executions 'dev' is set as NODE_ENV during spawn.
+   * @param fn
+   * @param params
+   */
+  useDevConfig (fn: Function, params: any): any {
+    const tmpEnv = process.env.NODE_ENV || 'default'
+    process.env.NODE_ENV = 'dev'
+    const rsp = fn.apply(this, params)
+    process.env.NODE_ENV = tmpEnv
+    return rsp
+  }
+
+  getServiceStackName (account: Account): string {
+    const tmpConfig = configModule.util.loadFileConfigs(path.join(process.cwd(), 'config'))
+    return `${MiraApp.getBaseStackNameFromParams(tmpConfig.app.prefix, tmpConfig.app.name, 'Service')}-${account.name}`
+  }
+
+  getAwsSdkConstruct (construct: any, account: Account): any {
+    const credentials = new aws.SharedIniFileCredentials({ profile: this.getProfile(this.env) || '' })
+    aws.config.credentials = credentials
+    // eslint-disable-next-line
+    // @ts-ignore
+    return new aws[construct]({ region: account.env.region })
+  }
+
+  async getFirstFailedNestedStackName (account: Account, stackName: string): Promise<string> {
+    const cloudformation = this.getAwsSdkConstruct('CloudFormation', account)
+    const events = await cloudformation.describeStackEvents({ StackName: stackName }).promise()
+    return events.StackEvents?.filter((event: StackEvent) => event.ResourceStatus === 'UPDATE_FAILED' || event.ResourceStatus === 'CREATE_FAILED')[0]?.PhysicalResourceId?.split('/')[1]
```

Comment:
We paired on this one, where i got an error like: "Error [ValidationError]: Stack [PaulDev-TestingStackErrors-Service-default-S3Webhosting0NestedStackS3Webhosting0Nested-1GS10YB3NDLZK] does not exist"
to fix, pass stack id and not name (maybe rename the function?) (AWS: You can list events for stacks that have failed to create or have been deleted by specifying the unique stack identifier (stack ID).)
to consider: console.log the error from try catch
console.log(chalk.red('Error, while getting error message from cloudformation. Seems something is wrong with your configuration.'))
*

Reply:
This has now been reported as #44 and will not be fixed as part of this PR/issue